test/unit/mwApiCapabilities.test.ts

```diff
@@ -0,0 +1,85 @@
+import MediaWiki from '../../src/MediaWiki.js'
+
+describe('Checking Mediawiki capabilities', () => {
```

Comment:
This test suit should have been introduced a while ago as a convenient way to check the capabilities of different wikis, as well as testing of the specific receipts. Before trying to scrape some wiki, this is a good place to start.